composer.json

```diff
@@ -82,11 +82,17 @@
         "spryker-feature/tax": "^202212.0",
         "spryker-shop/asset-widget": "^1.0.0",
```

Comment:
Are no tests needed on the project level?

Reply:
I did not get which tests you mean by commenting on the composer.json